### PR TITLE
KCP CLI: rename to kcp, documentation

### LINKED ISSUE #237: SKR CLI: open-source documentation
**Description**

Create documentation for SKR cli about general usage under https://github.com/kyma-project/control-plane/tree/master/docs/skr-cli

There should be an overview page, and subpages pointing to generated Markdown docs under docs/skr-cli similar to kyma cli gendocs or using the builtin cobra Markdown generator

For SAP Kyma Control Plane specific use, we will create internal doc with config files having the required params (OIDC client ID/secret, API URLs)

### PR DESCRIPTION
**Pull request by ebensom**
**Description**

Internal documentation for SKR CLI about general usage (not meant for kyma-project.io).

Changes proposed in this pull request:

- Rename to KCP CLI
- Add make target to generate SKR CLI commands docs
- Add KCP CLI overview and usage docs

**Related issue(s)**
Resolves #237

### CODE REVIEW

**Review by piotrmiskiewicz — APPROVED** [review]

File: docs/cli/commands/kcp_taskrun.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+## Examples
+
+```
+  kcp taskrun --target all kubectl patch deployment valid-deployment -p '{"spec":{"template":{"spec":{"containers":[{"name":"kubernetes-serve-hostname","image":"new image"}]}}}}'
+    Execute a kubectl patch operation for all runtimes
+  kcp taskrun --target account=CA4836781TID000000000123456789 /usr/local/bin/awesome-script.sh
+    Run a maintenance script for all runtimes of a given Global Account
+  kcp taskrun --target all helm upgrade -i -n kyma-system my-kyma-addon --values overrides.yaml
+    Deploy or a helm chart on all runtimes
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_taskrun.md:32]:
```suggestion
    Deploy a Helm chart on all runtimes
```

File: docs/cli/commands/kcp_taskrun.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+## Options
+
+```
+      --keep                         Keep downloaded kubeconfigs after execution for caching purpose
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_taskrun.md:38]:
```suggestion
      --keep                         Option that allows you to keep downloaded kubeconfig files after execution for caching purposes
```

File: docs/cli/commands/kcp_taskrun.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      --kubeconfig-dir string        Directory to download runtime kubeconfigs to. By default it is a random-generated directory in the OS specific default temporary directory (e.g. /tmp in Linux)
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_taskrun.md:39]:
```suggestion
      --kubeconfig-dir string        Directory to download Runtime kubeconfig files to. By default, it is a random-generated directory in the OS-specific default temporary directory (e.g. /tmp in Linux).
```

File: docs/cli/commands/kcp_taskrun.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  -p, --parallelism int              Number of parallel commands to execute (default 8)
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_taskrun.md:40]:
```suggestion
  -p, --parallelism int              Number of parallel commands to execute. The default value is 8.
```

File: docs/cli/commands/kcp_taskrun.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  -t, --target stringArray           List of runtime target specifiers to include (the option can be specified multiple times).
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_taskrun.md:41]:
```suggestion
  -t, --target stringArray           List of Runtime target specifiers to include. You can specify this option multiple times.
```

File: docs/cli/commands/kcp_taskrun.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+                                     A target specifier is a comma separated list of the following selectors:
+                                       all                 : all runtimes provisioned successfully and not deprovisioning
+                                       account=<REGEXP>    : Regex pattern to match against the runtime's GlobalAccount field. E.g. CA50125541TID000000000741207136, "CA.*"
+                                       subaccount=<REGEXP> : Regex pattern to match against the runtime's SubAccount field. E.g. 0d20e315-d0b4-48a2-9512-49bc8eb03cd1
+                                       region=<REGEXP>     : Regex pattern to match against the shoot cluster's Region field (provider region). E.g. "europe|eu-"
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_taskrun.md:46]:
```suggestion
                                       region=<REGEXP>     : Regex pattern to match against the Shoot cluster's region field (provider region), e.g. "europe|eu-"
```

File: docs/cli/commands/kcp_taskrun.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+```
+      --keep                         Keep downloaded kubeconfigs after execution for caching purpose
+      --kubeconfig-dir string        Directory to download runtime kubeconfigs to. By default it is a random-generated directory in the OS specific default temporary directory (e.g. /tmp in Linux)
+  -p, --parallelism int              Number of parallel commands to execute (default 8)
+  -t, --target stringArray           List of runtime target specifiers to include (the option can be specified multiple times).
+                                     A target specifier is a comma separated list of the following selectors:
+                                       all                 : all runtimes provisioned successfully and not deprovisioning
+                                       account=<REGEXP>    : Regex pattern to match against the runtime's GlobalAccount field. E.g. CA50125541TID000000000741207136, "CA.*"
+                                       subaccount=<REGEXP> : Regex pattern to match against the runtime's SubAccount field. E.g. 0d20e315-d0b4-48a2-9512-49bc8eb03cd1
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_taskrun.md:45]:
```suggestion
                                       subaccount=<REGEXP> : Regex pattern to match against the Runtime's subaccount field,  e.g. 0d20e315-d0b4-48a2-9512-49bc8eb03cd1
```

File: docs/cli/commands/kcp_taskrun.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+## Options
+
+```
+      --keep                         Keep downloaded kubeconfigs after execution for caching purpose
+      --kubeconfig-dir string        Directory to download runtime kubeconfigs to. By default it is a random-generated directory in the OS specific default temporary directory (e.g. /tmp in Linux)
+  -p, --parallelism int              Number of parallel commands to execute (default 8)
+  -t, --target stringArray           List of runtime target specifiers to include (the option can be specified multiple times).
+                                     A target specifier is a comma separated list of the following selectors:
+                                       all                 : all runtimes provisioned successfully and not deprovisioning
+                                       account=<REGEXP>    : Regex pattern to match against the runtime's GlobalAccount field. E.g. CA50125541TID000000000741207136, "CA.*"
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_taskrun.md:44]:
```suggestion
                                       account=<REGEXP>    : Regex pattern to match against the Runtime's global account field, e.g. CA50125541TID000000000741207136, "CA.*"
```

File: docs/cli/commands/kcp_taskrun.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+                                       subaccount=<REGEXP> : Regex pattern to match against the runtime's SubAccount field. E.g. 0d20e315-d0b4-48a2-9512-49bc8eb03cd1
+                                       region=<REGEXP>     : Regex pattern to match against the shoot cluster's Region field (provider region). E.g. "europe|eu-"
+                                       runtime-id=<ID>     : Runtime ID is used to indicate a specific runtime
+  -e, --target-exclude stringArray   List of runtime target specifiers to exclude (the option can be specified multiple times).
+                                     A target specifier is a comma separated list of the selectors described under --target option
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_taskrun.md:49]:
```suggestion
                                     A target specifier is a comma-separated list of the selectors described under the `--target` option.
```

File: docs/cli/commands/kcp_taskrun.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+## Options
+
+```
+      --keep                         Keep downloaded kubeconfigs after execution for caching purpose
+      --kubeconfig-dir string        Directory to download runtime kubeconfigs to. By default it is a random-generated directory in the OS specific default temporary directory (e.g. /tmp in Linux)
+  -p, --parallelism int              Number of parallel commands to execute (default 8)
+  -t, --target stringArray           List of runtime target specifiers to include (the option can be specified multiple times).
+                                     A target specifier is a comma separated list of the following selectors:
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_taskrun.md:42]:
```suggestion
                                     A target specifier is a comma-separated list of the following selectors:
```

File: docs/cli/commands/kcp_taskrun.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+                                       all                 : all runtimes provisioned successfully and not deprovisioning
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_taskrun.md:43]:
```suggestion
                                       all                 : All Runtimes provisioned successfully and not deprovisioning
```

File: docs/cli/commands/kcp_upgrade.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
+---
+title: kcp upgrade
+---
+Perform upgrade operations on Kyma runtimes
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_upgrade.md:4]:
```suggestion
Performs upgrade operations on Kyma Runtimes.
```

File: docs/cli/commands/kcp_upgrade_kyma.md [hunk 13]
```diff
@@ -0,0 +1,61 @@
+---
+title: kcp upgrade kyma
+---
+Upgrade or reconfigure Kyma on one or more Kyma runtimes
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_upgrade_kyma.md:4]:
```suggestion
Upgrades or reconfigures Kyma on one or more Kyma Runtimes.
```

File: docs/cli/commands/kcp_upgrade_kyma.md [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+  kcp upgrade kyma --target all --schedule maintenancewindow     Upgrade Kyma on all runtimes in their next respective maintenance window hours
+  kcp upgrade kyma --target "account=CA.*"                       Upgrade Kyma on runtimes of all Global Accounts starting with CA
+  kcp upgrade kyma --target all --target-exclude "account=CA.*"  Upgrade Kyma on runtimes of all Global Accounts not starting with CA
+  kcp upgrade kyma --target "region=europe|eu|uk"                Upgrade Kyma on runtimes whose region belongs to Europe
+```
+
+## Options
+
+```
+      --dry-run                      Perform the orchestration without executing the actual upgrage operations for the runtimes. The details can be obtained using the "kcp orchestrations" command
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_upgrade_kyma.md:29]: Please apply comments from the previous document's table here as well.

File: docs/cli/commands/kcp_upgrade_kyma.md [hunk 15]
```diff
@@ -0,0 +1,61 @@
+---
+title: kcp upgrade kyma
+---
+Upgrade or reconfigure Kyma on one or more Kyma runtimes
+
+## Synopsis
+
+Upgrades or reconfigures Kyma on targets of runtimes.
+The upgrade is performed by the Kyma Control plane within a new orchestration asynchronously, the id of which is returned by the command upon success.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_upgrade_kyma.md:9]:
```suggestion
The upgrade is performed by Kyma Control Plane (KCP) within a new orchestration asynchronously. The ID of the orchestration is returned by the command upon success.
```

File: docs/cli/commands/kcp_upgrade_kyma.md [hunk 16]
```diff
@@ -0,0 +1,61 @@
...
+The targets of runtimes are specified via the --target and --target-exclude options. At lease one --target must be specified.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_upgrade_kyma.md:10]:
```suggestion
The targets of Runtimes are specified using the `--target` and `--target-exclude` options. At least one `--target` must be specified.
```

File: docs/cli/commands/kcp_upgrade_kyma.md [hunk 17]
```diff
@@ -0,0 +1,61 @@
...
+The Kyma version and configurations to use for the upgrade is taken from the Kyma Control Plane during processing of the orchestration.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_upgrade_kyma.md:11]:
```suggestion
The Kyma version and configurations to use for the upgrade are taken from Kyma Control Plane during the processing of the orchestration.
```

File: docs/cli/commands/kcp_upgrade_kyma.md [hunk 18]
```diff
@@ -0,0 +1,61 @@
...
+
+```bash
+kcp upgrade kyma --target <TARGET SPEC> ... [--target-exclude <TARGET SPEC> ...] [flags]
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_upgrade_kyma.md:14]:
```suggestion
kcp upgrade kyma --target {TARGET SPEC} ... [--target-exclude {TARGET SPEC} ...] [flags]
```

File: docs/cli/01-01-overview.md [hunk 19]
```diff
@@ -0,0 +1,18 @@
...
+The kcp CLI (a.k.a. Kyma Control Plane CLI) is a day-two operations tool for Kyma runtimes, which allows to view and manage the runtimes in scale.
+It provides a set of commands you can use to:
+
+- List, filter and observe runtimes and their various attributes, states.
+- Download kubeconfig for runtimes.
+- Trigger orchestrations of Kyma upgrades and reconfigurations on targeted runtimes.
+- Display and filter orchestrations and their various attributes.
+- And more!
+
+The kcp CLI is not released, and may contain breaking changes compared to previous development versions. You can download the latest version from the development artifacts:
```
[the remark this answers is not in the document]
Comment (ebensom) [reply docs/cli/01-01-overview.md:14]: It means there is no released artifacts to download from github or kyma-release-artifacts bucket, like the case with kyma CLI. 
Both control-plane, and the KCP CLI is not released, we use and deploy the latest stable version.

File: docs/cli/01-01-overview.md [hunk 20]
```diff
@@ -0,0 +1,18 @@
...
+
+- Linux: [kcp-linux](https://storage.googleapis.com/kyma-development-artifacts/kcp/master/kcp-linux)
```
[the remark this answers is not in the document]
Comment (ebensom) [reply docs/cli/01-01-overview.md:16]: Yes, but the renaming of the CLI is also within this PR. Currently the binaries are still under skr* name.

File: docs/cli/02-01-usage.md [hunk 21]
```diff
@@ -0,0 +1,47 @@
...
+See [the full list of commands and flags](commands/kcp.md).
+
+|     Command        | Child commands   |  Description  | Example |
+|--------------------|----------------|---------------|---------|
+| [`kubeconfig`](commands/kcp_kubeconfig.md) | None | Downloads kubeconfig for given Kyma runtime. | `kcp kubeconfig -c a1fb2d35` |
+| [`login`](commands/kcp_login.md) | None | Performs OIDC login required by all commands. | `kcp login` |
+| [`orchestrations`](commands/kcp_orchestrations.md) | None | Displays Kyma control plane orchestrations and corresponding operations details. | `kcp orchestrations` |
+| [`runtimes`](commands/kcp_runtimes) | None | Displays Kyma runtimes based on various filters. | `kcp runtimes --region westeurope` |
+| [`taskrun`](commands/kcp_taskrun.md) | None | Runs generic tasks on one or more Kyma runtimes. | `kcp taskrun --target all kubectl get nodes` |
+| [`upgrade`](commands/kcp_upgrade.md) | [`kyma`](commands/kcp_upgrade_kyma.md) | Performs upgrade operations on Kyma runtimes. Currently only kyma upgrade is supported. | `kcp upgrade kyma --target all` |
```
[the remark this answers is not in the document]
Comment (ebensom) [reply docs/cli/02-01-usage.md:47]: Yes, shoot cluster upgrade.

File: docs/cli/commands/kcp.md [hunk 22]
```diff
@@ -0,0 +1,41 @@
+---
+title: kcp
```
[the remark this answers is not in the document]
Comment (ebensom) [reply docs/cli/commands/kcp.md:2]: I will move all detailed contents to here from usage. overview and usage will remain just really some starting page to those generated documents.

File: docs/cli/commands/kcp_orchestrations.md [hunk 23]
```diff
@@ -0,0 +1,51 @@
...
+  kcp orchestrations --state inprogress                                   Display all orchestrations which are in progress
+  kcp orchestration 0c4357f5-83e0-4b72-9472-49b5cd417c00                  Display details about a specific orchestration
+  kcp orchestration 0c4357f5-83e0-4b72-9472-49b5cd417c00 --operation OID  Display details of the specified runtime operation within the orchestration
+```
+
+## Options
+
+```
+      --operation string   Display details of the specified runtime operation when a given orchestration is selected
+  -o, --output string      Output type of displayed runtime(s). Possible values: table, json (default "table")
```
[the remark this answers is not in the document]
Comment (ebensom) [reply docs/cli/commands/kcp_orchestrations.md:30]: Default is auto-generated, cannot rephrase it like this. Will fix the rest.

File: docs/cli/commands/kcp_orchestrations.md [hunk 24]
```diff
@@ -0,0 +1,51 @@
...
+  -s, --state string       Filter output by state. Possible values: pending, inprogress, succeeded, failed
```
[the remark this answers is not in the document]
Comment (ebensom) [reply docs/cli/commands/kcp_orchestrations.md:31]: Values are literals parsed by the CLI, cannot rephrase.

File: docs/cli/commands/kcp_taskrun.md (hunk 4 above)
Comment (ebensom) [reply docs/cli/commands/kcp_taskrun.md:40]: default is auto-generated, cannot rephrase.

**Review by piotrmiskiewicz — APPROVED** [review]

File: docs/cli/commands/kcp_orchestrations.md [hunk 25]
```diff
@@ -0,0 +1,49 @@
+# kcp orchestrations
+Displays Kyma Control Plane (KCP) orchestrations.
+
+## Synopsis
+
+Display KCP orchestrations and their primary attributes, such as identifiers, type, state, parameters, or Runtime operations.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_orchestrations.md:6]:
```suggestion
Displays KCP orchestrations and their primary attributes, such as identifiers, type, state, parameters, or Runtime operations.
```

File: docs/cli/commands/kcp_orchestrations.md [hunk 26]
```diff
@@ -0,0 +1,49 @@
...
+The commands has two modes:
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_orchestrations.md:7]:
```suggestion
The command has two modes:
```

File: docs/cli/commands/kcp_orchestrations.md [hunk 27]
```diff
@@ -0,0 +1,49 @@
...
+  - Without specifying an orchestration ID as an argument, the command lists all orchestrations, or orchestrations matching the `--state` option, if provided.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_orchestrations.md:8]:
```suggestion
  - Without specifying an orchestration ID as an argument. In this mode, the command lists all orchestrations, or orchestrations matching the `--state` option, if provided.
```

File: docs/cli/commands/kcp_orchestrations.md [hunk 28]
```diff
@@ -0,0 +1,49 @@
...
+  - When specifying an orchestration ID as an argument, the command displays details about the specific orchestration.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_orchestrations.md:9]:
```suggestion
  - When specifying an orchestration ID as an argument. In this mode, the command displays details about the specific orchestration.
```

File: docs/cli/commands/kcp_orchestrations.md [hunk 29]
```diff
@@ -0,0 +1,49 @@
...
+     If the optional `--operation` is provided, it displays details of the specified Runtime operation within the orchestration.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_orchestrations.md:10]:
```suggestion
     If the optional `--operation` flag is provided, it displays details of the specified Runtime operation within the orchestration.
```

File: docs/cli/commands/kcp_runtimes.md [hunk 30]
```diff
@@ -0,0 +1,49 @@
+# kcp runtimes
+Displays Kyma Runtimes.
+
+## Synopsis
+
+Display Kyma Runtimes and their primary attributes, such as identifiers, region, or states.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_runtimes.md:6]:
```suggestion
Displays Kyma Runtimes and their primary attributes, such as identifiers, region, or states.
```

File: docs/cli/commands/kcp_taskrun.md [hunk 31]
```diff
@@ -0,0 +1,67 @@
...
+
+## Examples
+
+```
+  kcp taskrun --target all kubectl patch deployment valid-deployment -p '{"metadata":{"labels":{"my-label": "my-value"}}}'
+    Execute a kubectl patch operation for all Runtimes.
+  kcp taskrun --target account=CA4836781TID000000000123456789 /usr/local/bin/awesome-script.sh
+    Run a maintenance script for all Runtimes of a given global account.
+  kcp taskrun --target all helm upgrade -i -n kyma-system my-kyma-addon --values overrides.yaml
+    Deploy a helm chart on all Runtimes.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_taskrun.md:30]:
```suggestion
    Deploy a Helm chart on all Runtimes.
```

File: docs/cli/commands/kcp_taskrun.md [hunk 32]
```diff
@@ -0,0 +1,67 @@
...
+                                     A target specifier is a comma-separated list of the following selectors:
+                                       all                 : All Runtimes provisioned successfully and not deprovisioning
+                                       account=<REGEXP>    : Regex pattern to match against the Runtime's global account field, e.g. "CA50125541TID000000000741207136", "CA.*"
+                                       subaccount=<REGEXP> : Regex pattern to match against the Runtime's subaccount field, e.g. "0d20e315-d0b4-48a2-9512-49bc8eb03cd1"
+                                       region=<REGEXP>     : Regex pattern to match against the Runtime's provider region field, e.g. "europe|eu-"
+                                       runtime-id=<ID>     : Runtime ID is used to indicate a specific Runtime
+  -e, --target-exclude stringArray   List of runtime target specifiers to exclude. You can specify this option multiple times.
+                                     A target specifier is a comma-separated list of the selectors described under --target option.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_taskrun.md:47]:
```suggestion
                                     A target specifier is a comma-separated list of the selectors described under the --target option.
```

File: docs/cli/commands/kcp_taskrun.md [hunk 33]
```diff
@@ -0,0 +1,67 @@
...
+  -p, --parallelism int              Number of parallel commands to execute. (default 8)
+  -t, --target stringArray           List of Runtime target specifiers to include. You can specify this option multiple times.
+                                     A target specifier is a comma-separated list of the following selectors:
+                                       all                 : All Runtimes provisioned successfully and not deprovisioning
+                                       account=<REGEXP>    : Regex pattern to match against the Runtime's global account field, e.g. "CA50125541TID000000000741207136", "CA.*"
+                                       subaccount=<REGEXP> : Regex pattern to match against the Runtime's subaccount field, e.g. "0d20e315-d0b4-48a2-9512-49bc8eb03cd1"
+                                       region=<REGEXP>     : Regex pattern to match against the Runtime's provider region field, e.g. "europe|eu-"
+                                       runtime-id=<ID>     : Runtime ID is used to indicate a specific Runtime
+  -e, --target-exclude stringArray   List of runtime target specifiers to exclude. You can specify this option multiple times.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_taskrun.md:46]:
```suggestion
  -e, --target-exclude stringArray   List of Runtime target specifiers to exclude. You can specify this option multiple times.
```

File: docs/cli/commands/kcp_upgrade_kyma.md [hunk 34]
```diff
@@ -0,0 +1,59 @@
+# kcp upgrade kyma
+Upgrades or reconfigures Kyma on one or more Kyma Runtimes.
+
+## Synopsis
+
+Upgrades or reconfigures Kyma on targets of Runtimes.
+The upgrade is performed by Kyma Control Plane (KCP) within a new orchestration asynchronously. The ID of the orchestration is returned by the command upon success.
+The targets of Runtimes are specified via the `--target` and `--target-exclude` options. At least one `--target` must be specified.
+The Kyma version and configurations to use for the upgrade are taken from the Kyma Control Plane during the processing of the orchestration.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_upgrade_kyma.md:9]:
```suggestion
The Kyma version and configurations to use for the upgrade are taken from Kyma Control Plane during the processing of the orchestration.
```

File: docs/cli/commands/kcp_upgrade_kyma.md [hunk 35]
```diff
@@ -0,0 +1,59 @@
...
+                                     A target specifier is a comma-separated list of the following selectors:
+                                       all                 : All Runtimes provisioned successfully and not deprovisioning
+                                       account=<REGEXP>    : Regex pattern to match against the Runtime's global account field, e.g. "CA50125541TID000000000741207136", "CA.*"
+                                       subaccount=<REGEXP> : Regex pattern to match against the Runtime's subaccount field, e.g. "0d20e315-d0b4-48a2-9512-49bc8eb03cd1"
+                                       region=<REGEXP>     : Regex pattern to match against the Runtime's provider region field, e.g. "europe|eu-"
+                                       runtime-id=<ID>     : Runtime ID is used to indicate a specific Runtime
+  -e, --target-exclude stringArray   List of runtime target specifiers to exclude. You can specify this option multiple times.
+                                     A target specifier is a comma-separated list of the selectors described under --target option.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_upgrade_kyma.md:39]:
```suggestion
                                     A target specifier is a comma-separated list of the selectors described under the --target option.
```

File: docs/cli/commands/kcp_upgrade_kyma.md [hunk 36]
```diff
@@ -0,0 +1,59 @@
...
+  -t, --target stringArray           List of Runtime target specifiers to include. You can specify this option multiple times.
+                                     A target specifier is a comma-separated list of the following selectors:
+                                       all                 : All Runtimes provisioned successfully and not deprovisioning
+                                       account=<REGEXP>    : Regex pattern to match against the Runtime's global account field, e.g. "CA50125541TID000000000741207136", "CA.*"
+                                       subaccount=<REGEXP> : Regex pattern to match against the Runtime's subaccount field, e.g. "0d20e315-d0b4-48a2-9512-49bc8eb03cd1"
+                                       region=<REGEXP>     : Regex pattern to match against the Runtime's provider region field, e.g. "europe|eu-"
+                                       runtime-id=<ID>     : Runtime ID is used to indicate a specific Runtime
+  -e, --target-exclude stringArray   List of runtime target specifiers to exclude. You can specify this option multiple times.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_upgrade_kyma.md:38]:
```suggestion
  -e, --target-exclude stringArray   List of Runtime target specifiers to exclude. You can specify this option multiple times.
```

File: docs/cli/commands/kcp.md [hunk 37]
```diff
@@ -0,0 +1,40 @@
+# kcp
+Day-two operations tool for Kyma Runtimes
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp.md:2]:
```suggestion
Day-two operations tool for Kyma Runtimes.
```

File: docs/cli/commands/kcp.md [hunk 38]
```diff
@@ -0,0 +1,40 @@
...
+
+## Synopsis
+
+The KCP CLI (Kyma Control Plane CLI) is a day-two operations tool for Kyma Runtimes, which allows you to view and manage the Runtimes in scale.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp.md:6]:
```suggestion
KCP CLI (Kyma Control Plane CLI) is a day-two operations tool for Kyma Runtimes which allows you to view and manage the Runtimes in scale.
```

File: docs/cli/commands/kcp.md [hunk 39]
```diff
@@ -0,0 +1,40 @@
...
+It is possible to list and observe attributes and state of each Kyma Runtime and perform various operations on them, such as upgrading the Kyma version.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp.md:7]:
```suggestion
It is possible to list and observe attributes and state of each Kyma Runtime, and perform various operations on them, such as upgrading the Kyma version.
```

File: docs/cli/commands/kcp.md [hunk 40]
```diff
@@ -0,0 +1,40 @@
...
+You can find the complete list of possible operations as commands below.
+
+The CLI supports configuration file for common, global options needed for all commands. The config file will be looked up in this order:
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp.md:10]:
```suggestion
The CLI supports configuration file for common (global) options needed for all commands. The config file will be looked up in this order:
```

File: docs/cli/commands/kcp.md [hunk 41]
```diff
@@ -0,0 +1,40 @@
...
+  - `--config {PATH}` option
+  - KCPCONFIG environment variable which contains the path
+  - $HOME/.kcp/config.yaml (default path).
+
+The configuration file is in YAML format and supports the following global options: oidc-issuer-url, oidc-client-id, oidc-client-secret, keb-api-url, kubeconfig-api-url, gardener-kubeconfig.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp.md:15]:
```suggestion
The configuration file is in the YAML format and supports the following global options: `oidc-issuer-url`, `oidc-client-id`, `oidc-client-secret`, `keb-api-url`, `kubeconfig-api-url`, `gardener-kubeconfig`.
```

File: docs/cli/commands/kcp.md [hunk 42]
```diff
@@ -0,0 +1,40 @@
...
+Please see the Global Flags / Options section for the description of these options.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp.md:16]:
```suggestion
See the **Global Options** section of each command for the description of these options.
```

**Review by klaudiagrz — COMMENTED** [review]
Few more comments 😅

File: docs/cli/commands/kcp.md (hunk 41 above)
Comment (ebensom) [reply docs/cli/commands/kcp.md:15]: Unfortunately this part is generated from CLI global options list, and I can't easily apply Markdown syntax here.

**Review by klaudiagrz — APPROVED** [review]
LGTM 🦊

**Review by piotrmiskiewicz — APPROVED** [review]

**Pull request merged**